docs/9.x/changelog.md

```diff
@@ -56,6 +56,21 @@ extend updates past End of Support through customer agreements if required.
 
 9.0 is currently pre-release.
 
+### 9.0.0-beta.4 (August 31, 2021)
+
+#### Improvements
+
+* Generate prometheus-operator key pair to be used for metrics collection ([#2612](https://github.com/gravitational/gravity/pull/2612), [planet#858](https://github.com/gravitational/planet/pull/858)).
+
+#### Bugfixes
+
+* Fix a regression caused by the removal of tiller images from the gcr.io/kubernetes registry ([#2624](https://github.com/gravitational/gravity/pull/2624)).
+
+#### Internal Changes
+
+* Update CODEOWNERS ([#2625](https://github.com/gravitational/gravity/pull/2625)).
```

Comment:
I dunno that CODEOWNERS make the cut of what customers need to know about.  I might move `And other minor code cleanup and build improvements.` up under Improvements and skip Internal Changes this time around.